package/MDAnalysis/coordinates/base.py

```diff
@@ -1791,6 +1791,22 @@ def __init__(self, filename, convert_units=None, **kwargs):
 
         self._ts_kwargs = ts_kwargs
 
+    def copy(self):
+        """Return independent copy of this Reader.
+
+        New Reader will have its own file handle and can seek/iterate
+        independently of the original.
+
+        Will also copy the current state of the Timestep held in
+        the original Reader
+        """
+        new = self.__class__(self.filename,
+                             n_atoms=self.n_atoms)
+        new.ts = self.ts.copy()
```

Comment:
This will be confusing if the source reader isn't at frame 0 but some other frame *x*.  In such a case the new reader object will be at frame *0* but the stored timestep is from frame *x*.  Calling next will then lead to loading frame *1* instead of *x+1*.   You should seek the new reader to the frame of the old one to avoid this.

Reply:
A test for this behavior would also be good.